dev_requirements.txt

```diff
@@ -1,4 +1,4 @@
-
+Faker==7.0.1
```

Comment:
Faker is need in requirements.txt itself see [here](https://github.com/FlaskCon/traveller/blob/3e0fa7a568bd703db053a50e52692697e4ede718/traveller/modules/conf/upload.py) Forgot to precise ^^

Reply:
@jtrip